lisa/sut_orchestrator/libvirt/context.py

```diff
@@ -45,6 +45,7 @@ class NodeContext:
     next_disk_index: int = 0
     machine_type: Optional[str] = None
     enable_secure_boot: bool = False
+    use_cloud_init: bool = True
```

Comment:
Would it make sense to change this to a string enum named something like "init_system" with values "cloud-init" and "ignition". That way, it is easier to add another system in the future.